### ISSUE TITLE
[FEATURE REQ] should show email auto suggestion when typing email

### ISSUE THREAD

**Issue by kmsalmanrahman** [Cefalo/quick-meet#49]
As a user I want when I type email(s), should show  auto suggestion for good user experience

![image](https://github.com/user-attachments/assets/ee6a80e3-ac55-4630-831c-818e327cbf33)

**Comment by Propo41** [comment]
This is a good FR. The emails of all employees from the organization can be fetched and shown as an auto-suggestion dropdown like Google calendar does.

**Comment by imkurosaki** [comment]
@Propo41 hey, this is great so feature and i would like to know if i can possible to work on this?

**Comment by Propo41** [comment]
> @Propo41 hey, this is great so feature and i would like to know if i can possible to work on this?

You would need an organizational google account that utilizes meeting spaces in their calendars. If you don't have one, you won't be able to do this.

**Comment by imkurosaki** [comment]
Okay got it working on it, could you please add hacktoberfest label because it's my first time to participate on it.

**Comment by Propo41** [comment]
@imkurosaki do you want to work on this feature?

**Comment by imkurosaki** [comment]
Yes, i would love too but i don't have an organizational google account but you can assigned me i will find alternative or solution for this.

**Comment by Propo41** [comment]
@imkurosaki assigned you. This is a bit urgent task, would be nice if you could work on it sooner if time persists

**Comment by imkurosaki** [comment]
Hey @Propo41, I'm really sorry, but I won’t be able to work on this feature right now. I’m a bit busy with my thesis. Could we connect on LinkedIn?

**Comment by Propo41** [comment]
> Hey @Propo41, I'm really sorry, but I won’t be able to work on this feature right now. I’m a bit busy with my thesis. Could we connect on LinkedIn?

It's fine, I understand. Yeah sure, here's my linkedin: https://www.linkedin.com/in/ali-ahnaf/
